### PR TITLE
Refactoring of `AbstractSchema::getColumnPhpType()` method

### PR DESCRIPTION
**Pull request by Tigrov**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Schema/AbstractSchema.php [hunk 1]
```diff
@@ -402,36 +402,24 @@ protected function findTableNames(string $schema): array
...
     {
-        /** @psalm-var string[] $typeMap */
-        $typeMap = [
+        return match ($column->getType()) {
             // abstract type => php type
             SchemaInterface::TYPE_TINYINT => SchemaInterface::PHP_TYPE_INTEGER,
             SchemaInterface::TYPE_SMALLINT => SchemaInterface::PHP_TYPE_INTEGER,
-            SchemaInterface::TYPE_INTEGER => SchemaInterface::PHP_TYPE_INTEGER,
-            SchemaInterface::TYPE_BIGINT => SchemaInterface::PHP_TYPE_INTEGER,
+            SchemaInterface::TYPE_INTEGER => PHP_INT_SIZE === 4 && $column->isUnsigned()
```
Comment (terabytesoftw) [inline src/Schema/AbstractSchema.php:409]: This behavior is for all dbms or is specific,  i find it strange.

Comment (vjik) [reply src/Schema/AbstractSchema.php:409]: In PHP for 32bit system maximum integer number is `2147483647`. Seems, in most (maybe in all) DBMS unsigned int value will be more (for example in MySQL — `4294967295`).

**Review by vjik — APPROVED** [review]
Please, add line to changelog

**Comment by samdark** [comment]
👍

**Pull request merged**